mne_connectivity/spectral/epochs.py

```diff
@@ -634,14 +727,28 @@ def spectral_connectivity_epochs(
 
     Parameters
     ----------
-    data : array-like, shape=(n_epochs, n_signals, n_times) | Epochs
-        The data from which to compute connectivity. Note that it is also
-        possible to combine multiple signals by providing a list of tuples,
-        e.g., data = [(arr_0, stc_0), (arr_1, stc_1), (arr_2, stc_2)],
-        corresponds to 3 epochs, and arr_* could be an array with the same
-        number of time points as stc_*. The array-like object can also
-        be a list/generator of array, shape =(n_signals, n_times),
-        or a list/generator of SourceEstimate or VolSourceEstimate objects.
+    data : array-like, shape=(n_epochs, n_signals, n_times) | Epochs | ~mne.time_frequency.EpochsSpectrum
+        The data from which to compute connectivity. Can be epoched timeseries data as
+        an `array` or `~mne.Epochs` object, or Fourier coefficients for each epoch as an
+        `~mne.time_frequency.EpochsSpectrum` object. If timeseries data, the spectral
+        information will be computed according to the spectral estimation mode (see the
+        ``mode`` parameter). If an `~mne.time_frequency.EpochsSpectrum` object, this
+        spectral information will be used and the ``mode`` parameter will be ignored.
+
+        Note that it is also possible to combine multiple timeseries signals by
+        providing a list of tuples, e.g.: ::
+
+            data = [(arr_0, stc_0), (arr_1, stc_1), (arr_2, stc_2)]
+
+        which corresponds to 3 epochs where ``arr_*`` is an array with the same number
+        of time points as ``stc_*``. Data can also be a `list`/:term:`generator` of
+        arrays, ``shape (n_signals, n_times)``, or a `list`/:term:`generator` of
+        `~mne.SourceEstimate` or `~mne.VolSourceEstimate` objects.
+
+        .. versionchanged:: 0.8
+           Fourier coefficients stored in an `~mne.time_frequency.EpochsSpectrum` or
+           `~mne.time_frequency.EpochsSpectrumArray` object can also be passed in as
```

Comment:
At some point I think we decided to go back to being explicit about these refs since its behavior depends on `default_role` (and can silently fail to link). So here and elsewhere it would be better to do:
```suggestion
           Fourier coefficients stored in an :class:`~mne.time_frequency.EpochsSpectrum` or
           :class:`~mne.time_frequency.EpochsSpectrumArray` object can also be passed in as
```

Reply:
Okay, will add these.
Think I've also been lazy with the decomposition stuff as well, so will open an issue as a reminder to myself to fix.

Reply:
spec_conn_epochs should all be explicit now